docker/python-dill/build-and-push.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+for version in 3.6 3.6-slim 3.7 3.7-slim 3.8 3.8-slim
+do
+    python3 ../../../ci/jinja2_render.py '{"global":{"docker_prefix":'${DOCKER_PREFIX}'}}' python-dill/Dockerfile.$version python-dill/Dockerfile.$version.rendered
+    ../../docker-build.sh . Dockerfile.$version.rendered $(DOCKER_PREFIX)/hailgenetics/python-dill:$version
+    DOCKER_BUILDKIT=1 docker tag $(DOCKER_PREFIX)/hailgenetics/python-dill:$version docker.io/hailgenetics/python-dill:$version
```

Comment:
Doesn't this assume whomever uses this script has access to our dockerhub hailgenetics? Couldn't someone want to run this script in their own project and not push the image to docker hub? Or is this the purpose of putting it in build.yaml?

Reply:
Yeah, I think this should be hail-team only. Third parties should use build.yaml.